packages/components/src/components/badge/badge.stories.ts

```diff
@@ -1,14 +1,25 @@
 import '../../solid-components';
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, parameters } = storybookDefaults('sd-badge');
 const { overrideArgs } = storybookHelpers('sd-badge');
 const { generateTemplate } = storybookTemplate('sd-badge');
 
+/**
+ *
+ * Visual indicator of a quantity related to a particular element. Often displayed next to an icon or label.
```

Comment:
Used to visual indicate a quantity related to a particular element. Often displayed next to an icon or label.